practice-two/src/index.html

```diff
@@ -167,7 +167,82 @@ <h2 class="heading-section text-color-base testimonials__header-text">What Clien
     <!-- END TPROJECTS SECTION-->
 
     <!-- START REVIEW SECTION -->
-    <section></section>
+    <section class="review">
+      <div class="container review__wrapper">
+        <h2 class="heading-section text-color-secondary review__heading">Testimonials</h2>
+        <h2 class="heading-section text-color-secondary review__heading-desktop">What People Say</h2>
```

Comment:
The heading doesn't change based on the device, so please only choose a heading here. I think you should use the Testimonials heading